Makefile

```diff
@@ -31,10 +31,10 @@ update-compilers-image:
 	scripts/update-compilers-image.sh $(REGISTRY_PREFIX)
 
 bpftool-image: .buildx_builder
-	scripts/build-image.sh $(REGISTRY_PREFIX)/bpftool images/bpftool linux/amd64,linux/arm64 $(OUTPUT) "$$(cat .buildx_builder)"
+	scripts/build-image.sh $(REGISTRY_PREFIX)/cilium-bpftool images/bpftool linux/amd64,linux/arm64 $(OUTPUT) "$$(cat .buildx_builder)"
```

Comment:
If we rename these, do we need to adjust the consumer of these image names, ie the main dockerfile?

Reply:
No, I was meant to adjust these earlier in #3, but forgot.